server.go

```diff
@@ -936,3 +756,280 @@ func (s *Server) handlerHistory(w http.ResponseWriter, r *http.Request) {
 		s.l.Error("Error rendering template: %v", err)
 	}
 }
+
+func (s *Server) handleJikan(data *dataAddMovie, w http.ResponseWriter, r *http.Request, sourcelink string) ([]string, error) {
+
+	jikanEnabled, err := s.data.GetCfgBool("JikanEnabled", DefaultJikanEnabled)
+	if err != nil {
+		s.l.Error("Error while retriving config value 'JikanEnabled':\n %v", err)
+		s.doError(
+			http.StatusInternalServerError,
+			"something went wrong :C",
+			w, r)
+		return nil, err
```

Comment:
Instead of logging the error here and returning the bare error, return the wrapped error (the message that's currently being logged) and log the error in the calling context.  Same thing for the rest of the returns (including in `handleTmdb`).

Reply:
Will be changed in the next commit